### DISCUSSION TITLE
Installing only server

### DISCUSSION THREAD

**Discussion by AGluk** [SoftEtherVPN/SoftEtherVPN#1950] [Q&A]
Hello.

Please tell me, how to install only server without client and bridge.

Thank you!

**Comment by Evengard** [comment]
Depends on "from where" and "to where". Generally, your distro package system usually have separate packages for server and client.

If building from source, you can just delete whatever binary you don't need (except shared files of course, such as hamcore) - the actual executables aren't dependent on each other.

**Reply by AGluk** [reply]
Hello!

I'm trying to build and install it under Ubuntu 22.04 from the sources.

**make -C build** passed fine

The next command will install all components. But I need only server.

**Reply by Evengard** [reply]
I'm not sure we have something specific for that, as usually package manager maintainers do the install scripts themself. You either:

1. Just install everything as it is
2. Don't install at all and run from a directory of choice (maybe set up the systemd file manually)
3. Install everything and attempt to remove whatever isn't needed (`which vpnclient` and similar commands should help with that)
4. Consult the deb package sources on pointers how to do the correct install for your exact distro
5. Find some PPA which provides prebuilt packages of latest versions (I'm pretty sure there should be one) - there's the question of trust there though...

**Comment by AGluk** [comment]
I've tried to install it over

`apt install softether-vpnserver`

Good enough solution, but it doesn't install service.
Should I create it manually?

**Reply by Evengard** [reply]
That's weird. I'm pretty sure it should create a service file. I'd guess they named it differently, not like ours.

Also, distros don't install them into /etc/systemd/system, if that's where you'r trying to find it, but in a different place (/usr/lib/systemd/system if I'm not mistaken).

**Comment by AGluk** [comment]
Under Ubuntu 22.04 it calls **softether-vpnserver.service**.
So the commands should be:

```
systemctl enable softether-vpnserver
systemctl start softether-vpnserver
```

Now I can't find configuration file :(
I'd like to disable 443 port for listening. Only 5555.
Before it was under /vpnserver directory.

**Reply by Evengard** [reply]
Just do a `find / -name vpn_server.config`

**Comment by AGluk** [comment]
Man!

**/var/lib/softether/vpn_server.config**

I've tried to find vpnserver.config :( without underline symbol!
Thanks!!!

The last stupid question...
According to official instruction I should execute the following commands under HUB configuration:

```
SecureNatEnable
SstpEnable yes
```

But now it says:

`"SstpEnable": Command not found.`

Is there updated instruction for configuring SSTP server for the new version of softether?
Thanks in andvance!

**Reply by Evengard** [reply]
`ProtoOptionsSet SSTP /NAME:Enabled /VALUE:True`
Try that

**Reply by AGluk** [reply]
Thanks a lot.
It was enabled already. I've checked it by `ProtoOptionsGet SSTP`
Don't know why, but I can't open SSTP tunnel from my phone with the same configuration I used for v4 server.

IP_ADDRESS:5555
username@HUB_NAME
password

**Reply by Evengard** [reply]
SSTP works over port 443, I'm not sure it would through a different port.

Also, you may need to connect via a domain name, and also setup a valid SSL certificate (Let's Encrypt one works well)
